main.nf

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env nextflow
```

Comment:
The tests seem to be still not passing

Reply:
I was missing the update of the `conda.yml` but now that I've added them, conda is finding conflict of their dependencies. (Note that is works fine if you install them side by side without conda ...) 
Conda is a mess, [your original comment](https://github.com/EBIvariation/variant-remapping/pull/8#discussion_r433796517) of not using conda  at all sounds good right now  :face_with_head_bandage:

Reply:
Tests are now passing! 
I had to remove nextflow from the conda install and reinstate it into the github action workflow